packages/nest/README.md

```diff
@@ -25,11 +29,44 @@
 [OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API
 for feature flagging that works with your favorite feature flag management tool.
 
-🧪 This SDK is experimental.
+<!-- x-hide-in-docs-end -->
+
+## Overview
+
+The OpenFeature NestJS SDK is a package that provides a NestJS wrapper for the [OpenFeature Server SDK](https://openfeature.dev/docs/reference/technologies/server/javascript/).
+
+It's main capabilities are:
+
+- Provide a NestJS global module to simplify OpenFeature configuration and usage within NestJS;
+- Injecting feature flags directly into controller route handlers by using decorators;
+- Injecting transaction evaluation context for flag evaluations directly from [execution context](https://docs.nestjs.com/fundamentals/execution-context) (HTTP header values, client IPs, etc.);
+- Injecting OpenFeature clients into NestJS services and controllers by using decorators;
+- Setting up logging, event handling, hooks and providers directly when registering the module.
+
+## 🚀 Quick start
+
+### Requirements
+
+- Node.js version 16+
+- NestJS version 8+
+
+### Install
+
+#### npm
+
+```sh
+npm install --save @openfeature/nestjs-sdk
+```
+
+#### Required peer dependencies
+
+The OpenFeature SDK is required as peer dependency.
 
-#### Here's a basic example of how to use the OpenFeature NestJS API with `InMemoryProvider`.
+The minimum required version of `@openfeature/server-sdk` currently is `1.7.5`.
```

Comment:
We are missing rxjs and Nest here.

Reply:
Should `@nestjs/core` also be a peer dependency?

Reply:
Yes, this slipped through when we added the interceptors, before we only had `@nestjs/common`.
I will add that @beeme1mr.